pkg/util/util.go

```diff
@@ -836,18 +836,22 @@ func GetAndExtractZip(zipURL string, destination string) error {
 		return errors.Errorf("Invalid Zip URL: %s . Should either be prefixed with file://, http:// or https://", zipURL)
 	}
 
-	_, err := Unzip(pathToZip, destination)
+	filenames, err := Unzip(pathToZip, destination, pathToUnzip)
 	if err != nil {
 		return err
 	}
 
+	if len(filenames) == 0 {
+		return errors.Errorf("No files were unzipped, ensure that the project repo is not empty or that sparseCheckoutDir has a valid path")
```

Comment:
We shouldn't use `errors.Errorf()` as we don't use format string here, `errors.New()` should be sufficient.